.github/workflows/yamllint.yml

```diff
@@ -1,6 +1,6 @@
 ---
 name: "yamllint"
-on:
+"on":
```

Comment:
The change to double quotes of the hash key name is unnecessary here. If we are going to start quoting hash keys, we should either quote all or none of them.